lib/improver/nowcasting/lightning.py

```diff
@@ -186,7 +186,7 @@ def _update_metadata(cube):
         Modify the meta data of input cube to resemble a Nowcast of lightning
         probability.
 
-        1. Rename to "probability_of_lightning"
+        1. Rename to "probability_of_lightning_rate_above_threshold"
```

Comment:
@MoseleyS  Will this cause any problems with the old operational system (am I right the code is used in gpp?)